proto/dclupgrade/query.proto

```diff
@@ -21,6 +22,16 @@ service Query {
 		option (google.api.http).get = "/dcl/dclupgrade/proposed-upgrades";
 	}
 
+// Queries a ApprovedUpgrade by index.
+	rpc ApprovedUpgrade(QueryGetApprovedUpgradeRequest) returns (QueryGetApprovedUpgradeResponse) {
+		option (google.api.http).get = "/dcl/dclupgrade/approved_upgrade/{name}";
+	}
+
+	// Queries a list of ApprovedUpgrade items.
+	rpc ApprovedUpgradeAll(QueryAllApprovedUpgradeRequest) returns (QueryAllApprovedUpgradeResponse) {
+		option (google.api.http).get = "/dcl/dclupgrade/approved_upgrade";
```

Comment:
Please correct the endpoint (see the comment above).

Reply:
fixed

Reply:
Use dashes, not underscores.

Reply:
fixed